### ISSUE TITLE
Use of Angular setter getter

### ISSUE THREAD

**Issue by TheLarkInn** [rwwagner90/angular-styleguide-es6#1]
I think using the setter/getter can be permitted but only once at the entry point of ones app. With the ES6 module syntax you can load your components and pass the angular app variable as an argument. It makes your code really modular that way. 

Example from my ES6 Webpack Example:

``` javascript
const app = angular.module('app', ['ui.router']);

// Components Entrypoint
appComponents(app);

// Common Components Entrypoint
commonComponents(app);

// App Services Entrypoint
appServices(app);
```

**Comment by TheLarkInn** [comment]
https://github.com/TheLarkInn/angular-starter-es6-webpack/blob/master/app/index.js Is a starter app I wrote where that code comes from.

**Comment by RobbieTheWagner** [comment]
@TheLarkInn I do see the usefulness, in your specific case, but for people using the format in the guide, setters and getters should be avoided. I would be fine with documenting alternate cases, such as yours, but not sure we should remove that rule entirely.

**Comment by RobbieTheWagner** [comment]
@TheLarkInn what do you think? Should we document both?

**Comment by TheLarkInn** [comment]
@rwwagner90 do you see a case where anyone would ever need to use the setter/getter more than once per module? I think its worth discussion.

**Comment by RobbieTheWagner** [comment]
@TheLarkInn I'm not sure, but the point here is, if you don't need to pass your `app` into anything, you should just chain everything. The intent is to avoid unnecessary variables, but in this case the variable is necessary, so perhaps the wording should be changed to reflect something like "avoid unnecessarily using variables". Thoughts?

**Comment by TheLarkInn** [comment]
I agree, I think that's more on the right train of thought.

**Comment by RobbieTheWagner** [comment]
@TheLarkInn any chance you'd like to submit a PR correcting the language to fit both cases? I'd like to get the community contributing on this so we can create a nice collaborative doc :smile:

**Comment by RobbieTheWagner** [comment]
@TheLarkInn any chance you'd like to contribute? If not, I'll go ahead and try to incorporate what we discussed as best as I can.

**Comment by TheLarkInn** [comment]
I think I can manage something.

**Comment by RobbieTheWagner** [comment]
Merged the PR, please make a new issue and PR for the other things we were discussing.
